src/runmode-pfring.c

```diff
@@ -139,7 +139,7 @@ static void *OldParsePfringConfig(const char *iface)
     (void) SC_ATOMIC_ADD(pfconf->ref, pfconf->threads);
 
     if (strncmp(pfconf->iface, "zc", 2) == 0) {
-        SCLogInfo("ZC interface detected, not setting cluster-id");
+        SCLogInfo("%s: ZC interface detected, not setting cluster-id", pfconf->iface);
```

Comment:
think the backport should minimize user visible changes like the logging updates from master